Testscripts/Linux/validate-da.sh

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env bash
+#######################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# validate-da.sh
+# Description:
+#    Validate the installation and uninstallation of the Dependency Agent.
+#    Validate the enable and disable of the Dependency Agent.
+#######################################################################
+
+set -e
+set -x
+
+# Source utils.sh
+. utils.sh || {
+    echo "ERROR: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+
+# Get distro information
+GetDistro
+
+readonly dir_list=(/opt/microsoft/dependency-agent /var/opt/microsoft/dependency-agent /etc/opt/microsoft/dependency-agent /lib/microsoft-dependency-agent /etc/init.d/microsoft-dependency-agent)
+
+readonly da_pid_file="/etc/opt/microsoft/dependency-agent/config/DA_PID"
+readonly uninstaller="/opt/microsoft/dependency-agent/uninstall"
+
+function test_case_cleanup(){
+    [ -f "$uninstaller" ] && eval $uninstaller
+
+    rm -rf "${dir_list[@]}"
+
+    # This can be a NOP if the test exits before calling generate_network_activity
+    kill $network_activity_generator_pid
+
+    LogMsg "Finished cleaning up test cases"
+}
```

Comment:
Test suite clean up also needs to include removing the downloaded installer.

Reply:
Done